package.json

```diff
@@ -39,7 +40,7 @@
   "dependencies": {
     "broccoli-debug": "^0.6.5",
     "broccoli-kitchen-sink-helpers": "^0.3.1",
-    "broccoli-plugin": "^3.1.0",
+    "broccoli-plugin": "^4.0.1",
     "ensure-posix-path": "^1.0.2",
     "fast-sourcemap-concat": "^1.4.0",
```

Comment:
Do we not need to bump this to 2.1.0?

Reply:
Yes. We have to I did it locally didn't push that change. My bad. Updated it